### DISCUSSION TITLE
Visual Studio & Copilot

### DISCUSSION THREAD

**Discussion by Dew0Tgx** [community/community#70129] [Copilot Conversations]
Is it tru that Copilot only supports Visual Studio 22 ?

**Comment by Akash1134** [comment]
Hi @Dew0Tgx ,

Copilot is supported on multiple IDEs. Please [check this documentation](https://docs.github.com/en/copilot/getting-started-with-github-copilot?tool=azure_data_studio) to know more about the supported IDEs and getting started with Copilot.

Thanks!

**Comment by EdilandoSaturnino** [comment]
Hello Dew0Tgx,

GitHub Copilot is not built natively for the standard Visual Studio IDE.

However, GitHub has been continually working on enhancing and expanding the support for various platforms and editors. It's advisable to check [the official GitHub Copilot documentation](https://copilot.github.com) or GitHub's official announcements for the most recent updates regarding the support for Visual Studio 2022 or any other versions of Visual Studio.

**Comment by simoncalv** [comment]
[See some details here](https://learn.microsoft.com/en-us/visualstudio/ide/visual-studio-github-copilot-chat?view=vs-2022#Prerequisites) Copilot Chat is only supported in VS 2022, v17.6 onwards. Copilot "suggestions" latest, however are supported in VS 2022 17.5.5 [see here ](https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs) but you can grab earlier extension releases for prior VS versions.
